pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -36,9 +36,6 @@ import (
 )
 
 const (
-	// emptyUniquePodName is a UniquePodName for empty string.
-	emptyUniquePodName types.UniquePodName = types.UniquePodName("")
-
```

Comment:
I moved this into `types` package since there were a few places all defining UniquePodName(""). So it's still using it, just got rid of the duplication.